Makefile

```diff
@@ -0,0 +1,14 @@
+PYTHON ?= /usr/bin/python
+
+.PHONY: install
+install:
+	$(PYTHON) setup.py install
+
+.PHONY: clean
+clean:
+	$(PYTHON) setup.py clean --all
+
+.PHONY: test
+test:
+	pip install -r requirements.txt -r test/requirements.txt
```

Comment:
Installing requirements should probably be done with a separate make target, so it doesn't need to be executed with each test run.